package_manifest.yaml

```diff
@@ -601,25 +592,18 @@ katello_packages:
         - "el8-foreman-{{ foreman_version }}"
         - "el8-foreman-plugins-{{ foreman_version }}"
         - "el8-katello-candlepin-{{ katello_version }}"
-  children:
-    qpid_proton_packages: {}
   hosts:
-    jsoncpp: {}
     katello: {}
     katello-client-bootstrap: {}
     katello-repos: {}
     katello-selinux: {}
-    libwebsockets: {}
```

Comment:
can you doublecheck whether jsoncpp and websockets aren't used by mosquitto?

Reply:
These are in the katello packages list and therefore tag. While mosquitto is over in Foreman so there would be no crossover.

Reply:
Since you asked, not that I see. But you do need to make sure you provide systemd 3 times :laughing: 

```
/bin/sh
/bin/sh
/bin/sh
config(mosquitto) = 2.0.17-1.el8sat
libc.so.6()(64bit)
libc.so.6(GLIBC_2.14)(64bit)
libc.so.6(GLIBC_2.15)(64bit)
libc.so.6(GLIBC_2.17)(64bit)
libc.so.6(GLIBC_2.2.5)(64bit)
libc.so.6(GLIBC_2.3)(64bit)
libc.so.6(GLIBC_2.3.2)(64bit)
libc.so.6(GLIBC_2.3.4)(64bit)
libc.so.6(GLIBC_2.4)(64bit)
libcares.so.2()(64bit)
libcjson.so.1()(64bit)
libcrypto.so.1.1()(64bit)
libcrypto.so.1.1(OPENSSL_1_1_0)(64bit)
libdl.so.2()(64bit)
libdl.so.2(GLIBC_2.2.5)(64bit)
libm.so.6()(64bit)
libm.so.6(GLIBC_2.2.5)(64bit)
libmosquitto.so.1()(64bit)
libmosquitto.so.1(MOSQ_1.0)(64bit)
libmosquitto.so.1(MOSQ_1.1)(64bit)
libmosquitto.so.1(MOSQ_1.2)(64bit)
libmosquitto.so.1(MOSQ_1.3)(64bit)
libmosquitto.so.1(MOSQ_1.4)(64bit)
libmosquitto.so.1(MOSQ_1.5)(64bit)
libmosquitto.so.1(MOSQ_1.6)(64bit)
libmosquitto.so.1(MOSQ_1.7)(64bit)
libpthread.so.0()(64bit)
libpthread.so.0(GLIBC_2.12)(64bit)
libpthread.so.0(GLIBC_2.2.5)(64bit)
libssl.so.1.1()(64bit)
libssl.so.1.1(OPENSSL_1_1_0)(64bit)
libssl.so.1.1(OPENSSL_1_1_1)(64bit)
libstdc++.so.6()(64bit)
libstdc++.so.6(CXXABI_1.3)(64bit)
libstdc++.so.6(CXXABI_1.3.9)(64bit)
libsystemd.so.0()(64bit)
libsystemd.so.0(LIBSYSTEMD_209)(64bit)
rpmlib(CompressedFileNames) <= 3.0.4-1
rpmlib(FileDigests) <= 4.6.0-1
rpmlib(PayloadFilesHavePrefix) <= 4.0-1
rpmlib(PayloadIsXz) <= 5.2-1
rtld(GNU_HASH)
shadow-utils
systemd
systemd
systemd
```

Reply:
Right, just also looked at https://github.com/theforeman/foreman-packaging/blob/rpm/develop/packages/plugins/mosquitto/mosquitto.spec

It uses cjson, which of course is something completely different than jsoncpp.
And we disable websockets support.